README.md

```diff
@@ -75,6 +75,16 @@ The following settings are additionally recognised:
   if `True`, will delete any models in the reverse relation on the destination site that do not exist in the source site's reverse relation. As a result,
   it should only be used for models that behave strictly like child models but do not use `ParentalKey` - for example, tags, where importing an image with deleted tags should delete those tag linking models on the destination site as well.
 
+  Note: describing the relationship according to the format expected is important. An import may still complete succesfully if you've added a value that dosen't match, in which case the followed relation simply won't be updated and may cause unexpected problems on future imports. 
```

Comment:
I think there was also a bit of confusion around whether you specified the model to follow from, and the related_name, vs the model you want to follow to, and the field name. Worth adding a note about that here too (ie you could include it in your example, by saying if you want to import model Y from package X when importing Pages, linked via relationship name Z, then your setting would be...)